cheryl-cheng/SEPER
# Story/sep 5 suggest an article

**Summary**

- Did some whole app improvements: navigation, routing, styling
- Did some front end improvements to the existing suggestion form
- Set up the project backend and database connection
- Saving the data from the front end suggestions form into the backend 

**How to setup/test**

- Please create a file in the root directory ".env"
- In .env, create a variable called MONGO_URI equals to your own mongo db connection string, eg mine is 
MONGO_URI=mongodb+srv://maya:<password>@pappycluster.3kgfu.mongodb.net/<database_name>?retryWrites=true&w=majority 
- Ensure your env variable doesnt wrap the URI in ""
- Lastly, from the root directory (SEPER), run "npm start" which should boot up both front and back end
